resources/swapi/character.yaml

```diff
@@ -0,0 +1,110 @@
+"$schema": https://json-schema.org/draft/2020-12/schema
+"$id": character
+title: Character
+type: object
+required:
+- gid
+- id
+- label
+uniqueKeys:
+- - gid
+- - label
+  - id
+links:
+- rel: homeworld
+  href: homeworld/{id}
+  templateRequired:
+  - id
+  targetSchema:
+    "$ref": homeworld
+  templatePointers: id
+  targetHints:
+    directionality: outbound
+    multiplicity: has_one
+    backref: character
+- rel: films
+  href: films/{id}
+  templateRequired:
+  - id
+  targetSchema:
+    "$ref": films
+  templatePointers: id
+  targetHints:
+    directionality: outbound
+    multiplicity: has_one
+    backref: character
+- rel: species
+  href: species/{id}
+  templateRequired:
+  - id
+  targetSchema:
+    "$ref": species
+  templatePointers: id
+  targetHints:
+    directionality: outbound
+    multiplicity: has_one
+    backref: character
+- rel: starships
+  href: starships/{id}
+  templateRequired:
+  - id
+  targetSchema:
+    "$ref": starships
+  templatePointers: id
+  targetHints:
+    directionality: outbound
+    multiplicity: has_one
+    backref: character
+properties:
```

Comment:
homeworld should appear as a property

Reply:
If it is a link, should it be treated as a property as well? I was understanding that the links and properties are grouped separately on the schema

Reply:
Should links be appearing as properties and links? I thought it was one or the other.

Reply:
Nevermind homeworld should be in the properties now